styles/iconography/_styles.scss

```diff
@@ -47,6 +47,10 @@
         @include icon-small-block($fa-var-user);
     }
 
+    &.exclamation-triangle {
+        @include icon-small(before, right, $fa-var-exclamation-triangle);
```

Comment:
This looks correct, although you should really probably be using a second class to change the size.  `exclamation-triangle small-icon`